src/libmongoc/doc/mongoc_database_set_timeout_ms.rst

```diff
@@ -0,0 +1,31 @@
+:man_page: mongoc_database_set_timeout_ms
+
+mongoc_database_set_timeout_ms()
+================================
+
+Synopsis
+--------
+
+.. code-block:: c
+
+  bool
+  mongoc_database_set_timeout_ms (mongoc_database_t *database, int64_t timeout_ms, bson_error_t *error)
+
+Sets the client-side operation timeout value for ``database`` to the given ``timeout_ms``. If there is an error setting the timeout, this method will return false and ``error`` will be set.
+
+
+This function will set ``error`` and return false in the following cases:
+
+* ``database`` is NULL
```

Comment:
Remove, the setter asserts `database` is not NULL.

Reply:
Done.